app/components/primer/beta/truncate.pcss

```diff
@@ -0,0 +1,31 @@
+/* Truncate */
+
+.Truncate {
+  display: inline-flex;
+  min-width: 0;
+  max-width: 100%;
+
+  & > .Truncate-text {
+    min-width: 1ch;
+    max-width: fit-content;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    & + .Truncate-text {
```

Comment:
☝️ This selector will be slightly different

```diff
- .Truncate > .Truncate-text + .Truncate-text {
+ :is(.Truncate > .Truncate-text) + .Truncate-text {`
```

But browser support for `is()` should be ok and I _think_ specificity stays the same too. So maybe fine.